src/models/receipts.rs

```diff
@@ -36,39 +36,45 @@ pub struct ReceiptAction {
 }
 
 #[derive(Insertable)]
-#[table_name="actions_input_data"]
+#[table_name = "actions_input_data"]
```

Comment:
Should these be `receipt_actions_input_data`? Do you even need to name them explicitly? (I assumed that those are autogenerated)